src/docs/ftw-availability/availability-management/index.md

```diff
@@ -0,0 +1,138 @@
+availability window from `00:00` to `00:00`, enabling bookings that span
+entire days or overnight, the template does not have built-in support
+for creating hourly bookings that span multiple calendar days.
+
+### Availability exceptions
+
+Providers can set availability exceptions on listings to override the
+default plan for individual time ranges. Availability exceptions are
+created based on the unit type:
+
+- For daily bookings, exceptions are created per day, and the start and
+  end dates can be the same date
+- For nightly bookings, exceptions are created per night, and the end
+  date needs to be at least one day later than the start date
+- For hourly bookings and fixed-length bookings, exceptions are created
+  with a start time and an end time, and the end time needs to be at
+  least 5 minutes after the start time (both start and end time minutes
+  must be multiples of 5 and seconds and milliseconds must be 0)
+
+Regardless of the unit type, all exceptions are made using the same
+[endpoint](https://www.sharetribe.com/api-reference/marketplace.html#create-availability-exceptions).
+
+Providers can also block availability by using seats. Setting 0 seats
+for the duration of the availability exception sets the listing as
+unavailable for that period. If the period has existing bookings, the
+availability exception will not affect them, so it only blocks future
+bookings.
+
+Default availability, availability exceptions, and bookings are only
+visible to the provider of the listing. When other users view the
+listing’s availability on the listing page, they only see whether the
+listing is available, but they cannot see whether the availability
+results from the availability plan, exceptions, or existing bookings.
+
+## Listing search and timezone coverage
+
+The operator can allow users to search for listings based on their
+availability on certain dates. The date search cannot be restricted to
+specific time zones. This means that to cover listings from all possible
+time zones, the template prolongs the start and end times before sending
+them to the Sharetribe API. See the relevant code
+[in SearchPage.duck.js](https://github.com/sharetribe/web-template/blob/main/src/containers/SearchPage/SearchPage.duck.js#L185-L186).
+
+To be exact, the start time is moved 14 hours earlier, and the end time
+is moved 12 hours later. This means that the availability search results
+may include listings that are available before or after the time frame
+that the user enters.
+
+![Time range](time-range.png)
+
+## Filtering timeslot with intervals
+
+When querying timeslots for a listing using the booking unit "fixed" or
+"hourly", the template uses
+[interval-based filtering](/references/availability). This approach
+allows retrieving significantly less data from the API than using
+regular timeslot queries (the API is restricted to 500 timeslots per
+page). The query used in the template matches the first available
```

Comment:
Should we give the context: pagination?

```suggestion
regular timeslot queries (the API is restricted to 500 timeslots per paginated
page). The query used in the template matches the first available
```

Reply:
I would even expand the relevance of the pagination here:
- ...this approach allows retrieving significantly less data than regular time slot queries
- a regular timeslot query can fetch up to 500 time slots at once, and to fetch more you'd need to make multiple queries
- with short fixed time slots (e.g. 15 minutes), one day might have dozens of available time slots, so the first 500 time slots of a month might only cover a few weeks
- in the date picker, we're only interested in knowing whether there is availability on a certain day, so we make a query that matches the first available timeslot within the defined interval...

At least for me, it's not intuitive that 500 time slots is not enough for a monthly view of shorter time slots, so an explanation might make sense here